### PR TITLE
CLDR-16857 add SLE to 80, move SLL to 100

### PR DESCRIPTION
**Pull request by srl295**
- coverage

CLDR-16857

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by stenshamn — APPROVED** [review]
Looks right to me.

**Comment by AEApple** [comment]
Can we add a test that requires coverage to be added or explicitly opted out whenever we add a new currency?

**Comment by srl295** [comment]
@AEApple did just that

File: tools/cldr-code/src/test/java/org/unicode/cldr/util/TestLevel.java [hunk 1]
```diff
@@ -47,4 +55,49 @@ void testMax(final String a, final String b, final String c) {
...
+    }
+
+    @ParameterizedTest()
+    @MethodSource("modernCurrencies")
+    public void testModernCurrencies(final String code) {
+        Level l =
+                sdi.getCoverageLevel(
+                        String.format(
+                                "//ldml/numbers/currencies/currency[@type=\"%s\"]/symbol", code),
+                        "mt");
```
Comment (srl295) [inline tools/cldr-code/src/test/java/org/unicode/cldr/util/TestLevel.java:97]: proxy here because a language needs to be added, maybe it should be `und`?

**Review by pedberg-icu — APPROVED** [review]
Looks good to me

**Pull request merged**